frontend/src/views/RoleView.vue

```diff
@@ -69,10 +113,14 @@ const knowledgesStore = useKnowledgeStore()
 const createKnowledgeDialogFormVisible = ref(false)
 const role = ref<Role>()
 const knowledge = ref<Knowledge>({ weight: 0, name: '', roleId: 1 })
+const levelsStore = useLevelStore()
+const levels = ref<Level[]>()
```

Comment:
Do you need levels here? The levels are not being retrieved in the role already? (role.levels)